### PR TITLE
Migrating patient contact from hiv testing services

### PR DESCRIPTION
**Pull request by njorocs**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/kenyaemr/reporting/cohort/definition/evaluator/HTSFamilyContactsTestedCohortDefinitionEvaluator.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public EvaluatedCohort evaluate(CohortDefinition cohortDefinition, EvaluationCon
 
 		Cohort newCohort = new Cohort();
 
-		String qry="select c.id from kenyaemr_etl.etl_hts_contacts c where c.relationship_type in (970,971,972,1528,5617,162221)\n" +
+		String qry="select c.id from kenyaemr_etl.etl_hts_contacts c where c.relationship_type in (3,2,6,8)\n" +
```
Comment (ojwanganto) [inline api/src/main/java/org/openmrs/module/kenyaemr/reporting/cohort/definition/evaluator/HTSFamilyContactsTestedCohortDefinitionEvaluator.java:50]: We can join this to the relationship type and use the UUIDs. I am not sure if that will be unique though

File: api/src/main/java/org/openmrs/module/kenyaemr/chore/MigratePatientContactPersonAttributes.java [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+    /**
+     * @see AbstractChore#perform(PrintWriter)
+     */
+
+    @Override
+    public void perform(PrintWriter out) {
+
+        String insertPersonAttributes = "insert into person_attribute (person_id, value, person_attribute_type_id, creator, date_created, changed_by,\n" +
+                "                              date_changed, voided, voided_by, date_voided, void_reason, uuid)\n" +
```
Comment (ojwanganto) [inline api/src/main/java/org/openmrs/module/kenyaemr/chore/MigratePatientContactPersonAttributes.java:32]: This looks like a complex query. Wondering if using the module service to get contacts would have simplified the logic

File: api/src/main/java/org/openmrs/module/kenyaemr/reporting/cohort/definition/evaluator/HTSFamilyContactsTestedCohortDefinitionEvaluator.java (hunk 1 above)
Comment (njorocs) [reply api/src/main/java/org/openmrs/module/kenyaemr/reporting/cohort/definition/evaluator/HTSFamilyContactsTestedCohortDefinitionEvaluator.java:50]: Sure, it should be more safe to use uuids. Will make the changes across are refs.

**Review by patryllus — APPROVED** [review]
Looks okay - merging for tests

**Pull request merged**